langium/langium
# Added GoTo solver for properties in parser rules with return type

Partially addresses #425 

Adds GoTo functionality for properties in a parser rule referring to a property declared in an interface (only on parser rules that explicitly return a declared type).

How to test:
```
interface X {
    name: string
}

A returns X : name=ID;
```
Cmd+Click on `name` in the parser rule 'A' should send the cursor to the `name` property in interface X.


@msujew I finished the implementation before seeing your new commit with the new CstUtils. Maybe we could make use of the getParentChain function here.
